pkg/cvss/cvssv3/cvss.go

```diff
@@ -116,3 +150,54 @@ func Severity(score float32) storage.CVSSV3_Severity {
 	}
 	return storage.CVSSV3_UNKNOWN
 }
+
+// calculateScores calculates the CVSS v3.1 scores (Base, Exploitability and Impact)
+// based on the following guide: <https://www.first.org/cvss/v3.1/specification-document#Qualitative-Severity-Rating-Scale>
+func calculateScores(cvssV3 *storage.CVSSV3) {
```

Comment:
I recommend using https://github.com/stackrox/nvdtools/tree/master/cvss3 instead so we don't have to rewrite it